packages/search/tests/integration/components/navi-search-bar-test.js

```diff
@@ -13,14 +28,41 @@ module('Integration | Component | navi-search-bar', function(hooks) {
     await render(hbs`<NaviSearchBar />`);
 
     assert.equal(this.element.textContent.trim(), '');
+  });
+
+  test('perform search that returns results', async function(assert) {
+    await render(hbs`<NaviSearchBar />`);
```

Comment:
Can we add `assert.expect` statements for any test that has async behavior in it?